docs/blocks-stylesheet.md

```diff
@@ -103,3 +103,5 @@ add_action( 'enqueue_block_assets', 'gutenberg_boilerplate_es5_enqueue_common_as
 ```
 
 The `enqueue_block_assets` action is triggered both in the editor and on the front of the site. Since your block is likely to share some styles in both contexts, you can consider `style.css` as the base stylesheet, placing editor-specific styles in `editor.css`.
+
+> If you'd like the styles to be limited to the frontend only you can us `enqueue_block_assets` along with a call like `if ( ! is_admin() ) {}` etc.
```

Comment:
It's front end instead of frontend for nouns, see https://make.wordpress.org/core/handbook/best-practices/spelling/.